management/proto/management.proto

```diff
@@ -59,6 +63,47 @@ message ServerKeyResponse {
   google.protobuf.Timestamp expiresAt = 2;
 }
 
-message Empty {
+message Empty {}
+
+// WiretrusteeConfig is a common configuration of any Wiretrustee peer. It contains STUN, TURN, Signal and Management servers configurations
+message WiretrusteeConfig {
+  // a list of STUN servers
+  repeated HostConfig stuns = 1;
+  // a list of TURN servers
+  repeated HostConfig turns = 2;
+
+  // a Signal server config
+  HostConfig signal = 3;
+
+  // a Management server config
+  HostConfig management = 4;
+}
+
+// HostConfig describes connection properties of some server (e.g. STUN, TURN, Signal, Management)
+// User and Password are mostly used for TURN servers.
```

Comment:
To reduce confusion, I'd add 2 separate structs, like AnonymousHostConfig and LoggedInHostConfig (with better names, not these)

Reply:
I think we will need a protocol type to describe the connection with TURN/STUN servers (TLS, DTLS or PLAIN)

Reply:
Both done